resources/web/wwi/FloatingProtoParameterWindow.js

```diff
@@ -515,17 +535,20 @@ export default class FloatingProtoParameterWindow extends FloatingWindow {
   }
 
   #rotationOnChange(node) {
-    const object = {'x': node.inputs[0].value, 'y': node.inputs[1].value, 'z': node.inputs[2].value, 'a': node.inputs[3].value};
+    const object = {'x': parseFloat(node.inputs[0].value), 'y': parseFloat(node.inputs[1].value),
+      'z': parseFloat(node.inputs[2].value), 'a': parseFloat(node.inputs[3].value)};
```

Comment:
I got this warning: 
` Object properties must go on a new line if they aren't all on the same line. (object-property-newline)`